contracts/Strategy.sol

```diff
@@ -260,13 +266,12 @@ contract Strategy is BaseStrategy {
         uint256 amountToFree = _debtPayment.add(_profit);
 
         if (amountToFree > 0 && wantBal < amountToFree) {
-            (uint256 newLoose, ) = liquidatePosition(amountToFree.sub(wantBal));
+            (uint256 newLoose, ) = liquidatePosition(amountToFree);
 
             // if we didnt free enough money, prioritize paying down debt before taking profit
             if (newLoose < amountToFree) {
                 if (newLoose <= _debtPayment) {
                     _profit = 0;
-                    _loss += _debtPayment.sub(newLoose);
```

Comment:
what would happen in the case of a real loss? not a locked funds loss but a hack or bug that made kashi LPs to lose money? 

maybe is there a way to find out the reason we've not been able to withdraw full amount? (like checking available liquidity)

Reply:
If estimatedTotal assets has gone down relative to our debt, we have lost money. This will be indicated by kashi fraction exchange rate going down, so it should be accounted for.

Reply:
yeah I agree. I missed previous mention to loss, that you did not remove. all good